lib/friendly_shipping/services/usps_ship.rb

```diff
@@ -114,6 +116,32 @@ def rate_estimates(shipment, options:, debug: false)
         end
       end
 
+      # Get timing estimates.
+      # @see https://developer.usps.com/api/85#tag/Resources/operation/get-estimates API documentation
+      #
+      # @param shipment [Physical::Shipment] the shipment for which we want to get timings
+      # @param options [RateEstimateOptions] options for the timing estimate call
```

Comment:
Are these RateEstimateOptions or something else?

Reply:
Yes, they are `RateEstimateOptions` just because the rate estimates and timings share the same options. If they diverge in the future I'll add a new class specific to timings.

Unless you think it's worth adding a `TimingOptions` alias right now?

Reply:
I added a `TimingOptions` alias.